src/components/CampaignLocations/CampaignLocationsListItem.tsx

```diff
@@ -55,14 +56,14 @@ export const CampaignLocationsListItem: FunctionComponent<
             ID: {operatorId}
           </AppText>
         ) : null}
-        <AppText
-          style={[
-            styles.expiry,
-            hasExpired ? { color: color("grey", 40) } : {}
-          ]}
-        >
-          {hasExpired ? "Expired on " : "Valid till "}
-          {formattedExpiry}
+        <AppText style={[styles.expiry, { color: color("grey", 40) }]}>
+          {hasExpired
+            ? i18n.t("customerQuotaScreen.campaignExpiredOn", {
+                dateTime: formattedExpiry
+              })
+            : i18n.t("customerQuotaScreen.campaignValidTo", {
+                dateTime: formattedExpiry
+              })}
```

Comment:
Hmm, seems to have some duplication here. Any issue if we take `dateTime` param out of the translation template and do 

```
{hasExpired ?
    i18n.t("customerQuotaScreen.campaignExpiredOn") : i18n.t("customerQuotaScreen.campaignValidTo")}
{formattedExpiry}
```

Like the original logic?

Reply:
we can't take the dateTime param out because in chinese it's ` "campaignExpiredOn": "在 %{dateTime} 已到期",` and english it's `"campaignExpiredOn": "Expired on %{dateTime}",`